nautobot/core/api/fields.py

```diff
@@ -1,11 +1,14 @@
 from collections import OrderedDict
 
 from django.core.exceptions import ObjectDoesNotExist
+from drf_spectacular.utils import extend_schema_field
 from rest_framework import serializers
 from rest_framework.exceptions import ValidationError
 from rest_framework.relations import PrimaryKeyRelatedField, RelatedField
+from timezone_field.rest_framework import TimeZoneSerializerField as TimeZoneSerializerField_
 
 
+@extend_schema_field({"type": "object", "properties": {"value": {}, "label": {"type": "string"}}})
```

Comment:
It's not clear why this is necessary for this field. Can you please add a brief comment?

Reply:
That's cool and since this is only being done here at the serializer field it only needs to happen here. So that's not terrible, unless there is actually some way we could do it more centrally.